app/views/bill-runs/review-licence.njk

```diff
@@ -83,23 +83,23 @@
   <form method="post">
     <div class="govuk-button-group">
     {{ govukButton({
-      text: statusButtonText,
+      html: '<div data-test="status-button">' + statusButtonText + '</div>',
       classes: statusButtonClass,
       name: "licence-status",
       value: statusButtonValue,
       preventDoubleClick: true
     }) }}
 
     {{ govukButton({
-      text: progressButtonText,
+      html: '<div data-test="progress-button">' + progressButtonText + '</div>',
```

Comment:
The button component allows you to add an `attributes` option which is where the `data-test` thingy should go.